api/graphql/schema.graphql

```diff
@@ -131,6 +131,14 @@ type User {
   isProfileComplete: Boolean
   expectedGenericJobTitles: [GenericJobTitle] @belongsToMany
   priorityWeight: Int @rename(attribute: "priority_weight")
+
+  # teams and roles: built-in
+  roles: [Role!] @morphToMany
+  rolesTeams: [Team!] @morphToMany
+  permissionsTeams: [Permission!] @morphToMany
+  permissions: [Permission!] @morphToMany
```

Comment:
We don't need to expose all the relationships we have available. I kind of feel like we should just give them roleAssignments. But I'm open to leaving `roles` and adding a `teams` relationship (aliased to rolesTeams) if you feel its valuable.

Reply:
Removed in [remove extra relationships](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/69500c02baa6a090cdde1213656bb8672c78251c).